player/command.c

```diff
@@ -5298,6 +5298,7 @@ static void cmd_quit(void *p)
     if (write_watch_later || mpctx->opts->position_save_on_quit)
         mp_write_watch_later_conf(mpctx);
     mpctx->stop_play = PT_QUIT;
+    mpctx->quit_fast = true;
```

Comment:
the quit command is also reachable from libmpv, so this doesn't work

Reply:
How is terminating instance from cmd_quit useful for libmpv? 

Anyway, do you have good suggestions how to handle it?

Reply:
If you write an mpv gui frontend you may want to quit the player normally as an user would which is done via `quit` (what is this question?)

> Anyway, do you have good suggestions how to handle it?

AFAIK we currently share all source files between libmpv and mpv.
That could be changed so that at least once is compiled twice with different flags. Then via a global variable or whatever you want you'd be able to reliably tell whether this is mpv standalone or in library usage.

Of course this would mean frontend users still suffer from potential delays but we can't just introduce memory leaks into libmpv.

Reply:
> If you write an mpv gui frontend you may want to quit the player normally as an user would which is done via quit (what is this question?)

Yes, was just weird to terminate instance from inside of an instance without owner interaction, but I see there is `MPV_EVENT_SHUTDOWN` for this cases, so at lest there is a feedback.

> AFAIK we currently share all source files between libmpv and mpv.